README.md

```diff
@@ -78,6 +78,10 @@ const assistant = vga.VirtualGoogleAssistant.Builder()
     .create();
 ```
 
+### Proxy settings
+
+If you need to use a custom proxy, just set your `HTTP_PROXY` environment variable.
```

Comment:
To use and env variable makes it difficult to use this library as part of other solutions, for example we plan on adding it to bst tools so that you can use similar commands to alexa.
It should be added as part of the VirtualGoogleAssistantBuilder